src/aui/framemanager.cpp

```diff
@@ -3684,8 +3709,15 @@ void wxAuiManager::OnFloatingPaneResized(wxWindow* wnd, const wxRect& rect)
     // try to find the pane
     wxAuiPaneInfo& pane = GetPane(wnd);
     wxASSERT_MSG(pane.IsOk(), wxT("Pane window not found"));
+    // if frame isn't fully set up, don't stomp on pos/size info
+    if (!pane.frame)
+    {
+        return;
+    }
 
-    pane.FloatingSize(rect.GetWidth(), rect.GetHeight());
+    /* client size has precedence over wnd size,
+        so I don't think we need to set wnd size */
```

Comment:
Minor, but let's make this comment C++ and more authoritative:

```suggestion
    // Setting floating client size is enough, there is no need to set floating
    // size, as it won't be used if the client size is set.
```